assets/js/works.js

```diff
@@ -0,0 +1,87 @@
+const works = [
+  {
+    heading: 'Data Dashboard Healthcare',
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    languages: [
+      'html',
+      'bootstrap',
+      'ruby',
+    ],
+  },
+  {
+    heading: 'Website Portfolio',
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    languages: [
+      'html',
+      'bootstrap',
+      'ruby',
+    ],
+  },
+  {
+    heading: 'Professional Art Printing Data More',
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    languages: [
+      'html',
+      'bootstrap',
+      'ruby',
+    ],
+  },
+];
```

Comment:
- [ ] The site suppose to have 1 muti post project and 6 other project cards like the Figma design you just implement 3 projects only please add the other projects too and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/168063406-de394aa9-aedd-41db-88c1-12e52d508f08.png)  | ![image](https://user-images.githubusercontent.com/68796232/168063367-f926dc60-d0c0-4807-af83-e5c7c39aa5c3.png) |